modules/create_timepoint/jsx/createTimepointIndex.js

```diff
@@ -222,10 +222,22 @@ class CreateTimepoint extends React.Component {
     } else if (state.storage.visit[
         state.form.value.project
       ] !== undefined) {
-      state.form.options.visit = state.storage.visit[
-        state.form.value.project
-      ][state.form.value.subproject];
-      state.form.value.visit = null;
+      if (Array.isArray(state.storage.visit[
+        state.form.value.project][state.form.value.subproject])
+      ) {
+        const errorMessage = `No visit labels defined for subproject: ${
+          this.state.form.options.subproject[
```

Comment:
the visit labels are dependent on a project-subproject combination. so the error message should display both project and subproject accordingly

Reply:
@ridz1208 do you mean add the project name to the error message?